pkg/services/github.go

```diff
@@ -31,9 +31,11 @@ type GitHubOptions struct {
 }
 
 type GitHubNotification struct {
-	repoURL  string
-	revision string
-	Status   *GitHubStatus `json:"status,omitempty"`
+	RepoURLPath  string `json:"repoURLPath,omitempty"`
+	RevisionPath string `json:"revisionPath,omitempty"`
+	repoURL      string
```

Comment:
lets move private to the top of struct, as it was before

Reply:
DONE